blueprints/bootstrap-component/files/app/components/__name__.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-bootstrap-controls/components/<%= dasherizedModuleName %>';
```

Comment:
Is this real?

Reply:
Yes. That is an ember blueprint file. They are used by the generators.